src/fragments/lib/analytics/js/getting-started.mdx

```diff
@@ -171,15 +171,19 @@ Amazon Pinpoint service requires an IAM policy in order to use the `record` API:
 ## Update Endpoint
 
 An endpoint uniquely identifies your app within Pinpoint. In order to update your <a href="https://docs.aws.amazon.com/pinpoint/latest/apireference/rest-api-endpoints.html" target="_blank">endpoint</a> use the `updateEndpoint()` method:
+Know that `updateEndpoint` method takes the object containing the configuration of `Analytics.AWSPinpoint.endpoint`, not the whole configuration object used to [set up the existing analytics backend](#set-up-existing-analytics-backend).
```

Comment:
```suggestion
```
Though I definitely understand adding this, I think that we accomplish the same thing below by adding your "optional" note below. All of the possible attributes are also documented below.

Reply:
Most important:
Am I able to use `updateEndpoint` to update for example: `Auth.region` or `Analytics.autoSessionRecord`?

If yes then right - we can remove this note, but if not, and if only the `Analytics.AWSPinpoint.endpoint` can be updated, then I would keep this note to reduce the confusion as much as possible.

Reply:
@david-mcafee  Just a kind reminder in case my question went unnoticed ⬆️

Reply:
@david-mcafee What is your opinion about my concerns?